app.js

```diff
@@ -35,15 +35,25 @@ const { getUserOrCreateUser } = require("./api/helpers/user.js");
 
 const port = process.env.PORT || 3001;
 
+// canonicalize url
+app.use((req, res, next) => {
+  if (
+    process.env.NODE_ENV === "production" &&
+    req.hostname !== "participedia.net"
+  ) {
+    res.redirect("https://participedia.net" + req.originalUrl);
+  }
+  next();
+});
 // CONFIGS
 app.use(compression());
 app.set("port", port);
 app.use(express.static("public", { index: false }));
 app.use(morgan("dev")); // request logging
 app.use(methodOverride()); // Do we actually use/need this?
 app.use(cors());
-app.use(bodyParser.json({ limit: "50mb" }));
```

Comment:
this was changed to allow for multiple files to be uploaded at once. do you see an issue with keeping this at 50mb?